pkg/core/resources/apis/mesh/dataplane_helpers_test.go

```diff
@@ -373,7 +373,18 @@ var _ = Describe("Dataplane", func() {
 `,
 				expected: "",
 			}),
-			Entry("dataplane with 1 inbound interface", testCase{
+			Entry("dataplane with address in networking", testCase{
+				dataplane: `
+                networking:
+                  address: 192.168.0.1
+                  inbound:
+                  - port: 8080
```

Comment:
Should we also set `networking.inbound[].address` ? To make sure that `networking.address` is chosen over it